qa/setup_packages.py

```diff
@@ -316,29 +316,24 @@ def get_pyvers_name(self, url, cuda_version):
 
 all_packages = [PlainPackage("opencv-python", ["4.2.0.32"]),
                 CudaPackage("cupy",
-                        { "90"  : ["7.3.0"],
-                          "100" : ["7.3.0"] },
+                        { "100" : ["7.5.0"] },
                         "cupy-cuda{cuda_v}"),
                 CudaPackage("mxnet",
                         { "90"  : ["1.6.0"],
```

Comment:
Probably this also should be removed here, right?

Reply:
Missed that. Done